apps/src/templates/instructions/InlineFeedback.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Radium from 'radium';
+
+const InlineFeedback = (props) => {
+  return (
+    <div style={props.style.container}>
+      <p style={props.style.message}>{props.message}</p>
+      {props.extra && <p style={props.style.message}>{props.extra}</p>}
+    </div>
+  );
+};
+
+InlineFeedback.propTypes = {
+  messge: React.PropTypes.string.isRequired,
+  extra: React.PropTypes.string,
```

Comment:
would be good to document the style prop here. Also I might call it `styles` instead of `style` to differentiate it from a standard style prop.